Example/RelayIntegrationTests/RelayClientEndToEndTests.swift

```diff
@@ -74,38 +61,50 @@ final class RelayClientEndToEndTests: XCTestCase {
         super.tearDown()
     }
 
-//    func testSubscribe() {
-//        relayA = makeRelayClient(prefix: "")
-//
-//        do {
-//            try relayA.connect()
-//        } catch {
-//            XCTFail("Failed to connect: \(error)")
-//        }
-//
-//        let subscribeExpectation = expectation(description: "subscribe call succeeds")
-//        subscribeExpectation.assertForOverFulfill = true
-//        relayA.socketConnectionStatusPublisher.sink { [weak self] status in
-//            guard let self = self else {return}
-//            if status == .connected {
-//                Task(priority: .high) {  try await self.relayA.subscribe(topic: "ecb78f2df880c43d3418ddbf871092b847801932e21765b250cc50b9e96a9131") }
-//                subscribeExpectation.fulfill()
-//            }
-//        }.store(in: &publishers)
-//
-//        wait(for: [subscribeExpectation], timeout: InputConfig.defaultTimeout)
-//    }
-
-    func testEndToEndPayload() {
+    // test_bundleId_present - configured in the cloud to include bundleId for whitelisted apps
```

Comment:
what about the failure case? Use a bundleId that is not added to the Cloud allowed ids

Reply:
I think this should be covered by the relay